ca/ca.go

```diff
@@ -101,6 +101,13 @@ type certificateStorage interface {
 	AddCertificate(context.Context, []byte, int64, []byte) (string, error)
 }
 
+type certificateType string
+
+const (
+	precertificate = certificateType("precertificate")
+	certificate    = certificateType("certificate")
```

Comment:
Let's name these enums `precertType` and `certType`. Otherwise they look like variables holding a precertificate (or a certificate) in the calls below.